pom.xml

```diff
@@ -1401,11 +1403,12 @@ flexible messaging model and an intuitive client API.</description>
           <executions>
               <execution>
                   <id>source-release-assembly-tar-gz</id>
-                  <phase>generate-sources</phase>
+                  <phase>package</phase>
```

Comment:
this is not good, we have to capture sources before building, otherwise we could pickup some unwanted stuff.

IIRC I had to set generate-sources also because some downstream modules (docker image for CPP build?) needed the source tarbal

Reply:
sure, I'll put it back to generate-sources. I found a way to skip this by adding a property. I needed to add `<inherit>false</inherit>` to get it working properly.